src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.css

```diff
@@ -21,3 +23,14 @@
 .input-icon {
   composes: input-icon from '../custom-field-input-text/custom-field-input-text.css';
 }
+
+.popup-container {
+  composes: container from '../listbox/listbox.css';
+  background-color: var(--white);
+  box-sizing: border-box;
+  max-height: 160px;
+  overflow: scroll;
+  position: absolute;
+  width: 100%;
+  z-index: 10;
```

Comment:
I can see this being a problem depending on the page it's used in. We should probably tackle documenting z-indices for different use-cases as part of our design consistency effort, but for now, I wonder if we should just set it to `2` and let people compose/override for their particular case?

Reply:
`2` doesn't work with React Styleguidist. :( I think it sets a z-index of like 5 or something on each code example so it renders weird. :(

Reply:
Oh damn, that's unfortunate. Let's leave it for now, then.